newyork.html

```diff
@@ -64,27 +75,27 @@ <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
 
         <div >
           <div >
-            <img class="image1" alt src="18cosmos-superJumbo.jpg">
-            <p>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...Rick Friedman for The New York Times</p>
+            <img class="image1" alt src="images/18cosmos-superJumbo.jpg">
+            <p class="main-article">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...Rick Friedman for The New York Times</p>
             
           </div>
 
```

Comment:
## Improvement on Page's navbar & main image styling as seen on large screens. 
- [ ] The original page has its elements in the right, middle and left bordered evenly to give a sense of enough space on the far end of the page's `navbar`. In our case, we have the icons on the left and the buttons on the right less of space as expected on the borders of the page. Kindly increase the border space of these elements. There need to be space between these elements and the page's edge.
- [ ] The page's main image seems to be bigger than expected. Would you mind reducing its size?
- [ ] The main article's heading is styled in such a way that the workings are wrapping on the next line at the `Bang's` word and the words are indented too. The social icons beneath it are also lighter in colour. There is also the horizontal line that should be much thinner than what we currently have and much shorter as well. Would you mind making these changes so as the clone page resemble the original page in this aspect?

- Below are the comparisons you can use as a guide.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|![main-image-header](https://user-images.githubusercontent.com/22345961/108571056-5c52a580-7320-11eb-8d5b-afef438add61.png)|![main-image-header-clone](https://user-images.githubusercontent.com/22345961/108571065-6379b380-7320-11eb-828b-78b9f0e5e243.png)|